scss/ui/partials/ui/onboarding/_ngonboarding.scss

```diff
@@ -32,10 +32,8 @@
 }
 
 .onboarding-popover.onboarding-centered {
-  // sass-lint:disable no-vendor-prefixes
-  -moz-transform: translate(-50%, -50%);
-  -ms-transform: translate(-50%, -50%);
-  -webkit-transform: translate(-50%, -50%);
+  // This can be removed after we drop support for Chrome before version 36.
+  -webkit-transform: translate(-50%, -50%); // sass-lint:disable-line no-vendor-prefixes
```

Comment:
We should be able to replace this with the [Compass translate mixin](http://compass-style.org/reference/compass/css3/transform/#mixin-translate), which will add vendor prefixes as configured by the SCSS build.